tests/test_response/test_user_response.py

```diff
@@ -125,3 +126,13 @@ def test_filter_response_by_invalid_room_id(self):
             filter_question_by_invalid_room,
             filter_question_by_invalid_room_response
         )
+
+    def test_response_for_wrong_question_type(self):
+        """
+        Testing for responding to a question with the wrong question type
```

Comment:
Consider having this docstring as something like `Test that a user cannot respond to a question with the wrong question type`